lnbits/core/views/generic.py

```diff
@@ -41,7 +41,7 @@
 
 @generic_router.get("/favicon.ico", response_class=FileResponse)
 async def favicon():
-    return FileResponse("lnbits/core/static/favicon.ico")
+    return FileResponse("lnbits/static/favicon.ico")
```

Comment:
- do not use OS dependent path separator
```suggestion
    return FileResponse(Path("lnbits", "static", "favicon.ico"))
```
- also must add `from pathlib import Path`